reconcile/utils/ocm_base_client.py

```diff
@@ -0,0 +1,104 @@
+import logging
+from typing import Any, Mapping, Optional
+from requests import Session, codes
+
+from sretoolbox.utils import retry
+
+
+REQUEST_TIMEOUT_SEC = 60
```

Comment:
i love the fact, that we have timeouts, i just wonder if 60 seconds is too high considering that the `init_access_token` function uses it while having the `@retry` decorator which does 3 attempts per default.

i know that you just moved this code here so i will accept an "not the scope of this change" comment if you should give it :D

Reply:
maybe you should ignore my nit. that way this change is functionally neutral! go merge